src/api/catalog-search/catalog-search.ts

```diff
@@ -182,10 +183,11 @@ export class CatalogSearchAPI {
     query?: string;
     filters?: CatalogSearchFilters;
     sort?: CatalogSearchSort;
+    exactMatch?: boolean;
```

Comment:
I don't think this is the way to go. IMO it would make more sense to implement this as a standalone method rather than expanding the surface area of the `search` api and related hooks. I.E `CatalogSearchAPI.findByName`

This would be exposed to our UI code
via `useSearchContext` like so:

```tsx
const searchAPI = useSearchContext();

const versions = searchAPI.findByName("@aws-cdk/core")
```

Reply:
Agreed, thanks for the suggestion.